app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
```

Comment:
download policy should be nil for repos that aren't yum repos as far as I can tell

```
2.2.4 :017 > docker_repo = Katello::Repository.where(:name => "docker1").first
2.2.4 :018 > docker_repo.download_policy
```